### ISSUE TITLE
What about creating a app to interact with DAI Maker?

### ISSUE THREAD

**Issue by danimesq** [empowerthedao/template-aragon-agent-app#1]
So, DAOs could open CDPs and use democracy to know the best decisions when lending and re-investing

**Comment by danimesq** [comment]
@willjgriff

**Comment by willjgriff** [comment]
This was part of our original nest proposal but our proposal was reduced, and we removed this, as there wasn't a sufficient budget available.

**Comment by danimesq** [comment]
I don't think this repo is appropriate for this issue. I suggest creating a repo called "meta", where it can be suggested new apps and directions for the ETD project. Then, you can use the GitHub move feature for moving this issue to such repo.
